tests/std/tests/P0896R4_views_drop_while/test.cpp

```diff
@@ -0,0 +1,431 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
Should this also `test_cpo` in `tests/std/tests/P0896R4_ranges_range_machinery/test.cpp`?